manifests/i/isnetwork/ISNAutoItStudio/1.13/isnetwork.ISNAutoItStudio.locale.de-DE.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: isnetwork.ISNAutoItStudio
+PackageVersion: 1.13
+PackageLocale: de-DE
+Publisher: isnetwork
+PublisherUrl: https://www.isnetwork.at
+PublisherSupportUrl: https://www.isnetwork.at
+PrivacyUrl: https://www.isnetwork.at/datenschutz
+Author: ISI360
+PackageName: ISN AutoIt Studio
+PackageUrl: https://www.isnetwork.at
+License: GNU General Public License
+LicenseUrl: https://www.isnetwork.at
+ShortDescription: AutoIt IDE

```

Please add a better Description.

```suggestion
ShortDescription: AutoIt IDE
```